src/cli/index.js

```diff
@@ -220,9 +222,17 @@ const cli = () => {
         cachedBundle = updateBundle();
       }); // initial cache
 
+      const redoclyClient = new RedoclyClient();
+      const isRedoclyUser = redoclyClient.hasToken() && !!(await redoclyClient.getAuthorizationHeader());
```

Comment:
could you move this to `redoclyClient.isAuthorize()`. Also, MB rename to `isAuthorizedWithRedocly`

Reply:
also, can you calculate `useRedocPro` here instead? If `useCommunityEdition` is `true` we don't need to try to authorize with redocly.

Also, `useRedocPro` make more sense as an arg to `startPreviewServer` and `getPageHTML`